### ISSUE TITLE
New vision/eye MP FO-wave implicit time

### ISSUE THREAD

**Issue by abby-jonesKOMP** [obophenotype/mammalian-phenotype-ontology#3347]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label: abnormal FO-wave implicit time

>2. Synonyms: n/a

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

anomaly in the time from the onset of light stimulus to the peak of the FO-wave

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal ERG implicit time (MP:0012033)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

**Comment by sbello** [comment]
abnormal - MP:0020978
increased - MP:0020980
decreased - MP:0020979
